config/packages/easy_admin.yaml

```diff
@@ -33,20 +31,9 @@ easy_admin:
                 - ckeditor_styles.css
                 - styles.css
     entities:
-        ThemeCategory:
-            item_permission: ['ROLE_ADMIN', 'ROLE_GROUP_ADMIN']
-            class: App\Entity\ThemeCategory
-            list:
-                fields:
-                    - { property: 'sort_order', label: 'entity.theme_category.sort_order' }
-                    - { property: 'category', label: 'entity.theme_category.category' }
-                    - { property: 'theme', label: 'entity.theme_category.theme' }
-                sort: ['sortOrder', 'ASC']
-                actions: []
-
         Theme:
             class: App\Entity\Theme
-            item_permission: ['ROLE_ADMIN', 'ROLE_GROUP_ADMIN']
+            item_permission: ['ROLE_SUPER_ADMIN', 'ROLE_ADMIN', 'ROLE_GROUP_ADMIN']
```

Comment:
I'm not sure we should check for `ROLE_SUPER_ADMIN` here (and in other places).
As per [the role hierarchy](https://github.com/aakb/itstyr/blob/develop/config/packages/security.yaml#L7) `ROLE_SUPER_ADMIN` implies `SUPER_ADMIN`, and `ROLE_SUPER_ADMIN` should probably only be used if we really need to check for superpower.